retrofit2-clients/src/main/java/com/palantir/remoting1/retrofit2/Retrofit2ClientBuilder.java

```diff
@@ -50,15 +52,27 @@ public Retrofit2ClientBuilder(ClientConfig config) {
     @Override
     public <T> T build(Class<T> serviceClass, String userAgent, List<String> uris) {
```

Comment:
Retrofit2 throws if you don't give it /-terminated base URLs. So there's no ambiguity in behaviour.